e2e-tests/tests/one-pod/01-create-cluster.yaml

```diff
@@ -22,6 +22,3 @@ commands:
         | yq eval '.spec.backup.storages.minio.s3.endpointUrl="http://minio-service:9000"' - \
         | yq eval '.spec.backup.storages.minio.s3.region="us-east-1"' - \
         | kubectl -n "${NAMESPACE}" apply -f -
-
-      wait_cluster_consistency "${test_name}" "1" "1"
-    timeout: 240
```

Comment:
Why did we delete it?

Reply:
Because in `01-assert.yaml` we wait for 240 sec for cluster to get ready status, which is basically the same as this wait_cluster_consistency.